applications/luci-app-wireguard/htdocs/luci-static/resources/view/wireguard/status.js

```diff
@@ -0,0 +1,223 @@
+'use strict';
+'require view';
+'require rpc';
+'require form';
+'require poll';
+
+
+var callGetWgInstances = rpc.declare({
+	object: 'luci.wireguard',
+	method: 'getWgInstances'
+});
+
+
+function bytesToStr(bytes) {
+	bytes = parseFloat(bytes);
+	if (bytes < 1) { return '0 B'; }
+	var sizes = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB'];
+	var i = parseInt(Math.floor(Math.log(bytes) / Math.log(1024)));
+	return Math.round(bytes / Math.pow(1024, i), 2) + ' ' + sizes[i];
+}
+
+function timestampToStr(timestamp) {
+	if (timestamp < 1) {
+		return _('Never');
+	}
+	var now = new Date();
+	var seconds = (now.getTime() / 1000) - timestamp;
+	var ago = '';
+	if (seconds < 60) {
+		ago = parseInt(seconds) + _('s ago');
+	} else if (seconds < 3600) {
+		ago = parseInt(seconds / 60) + _('m ago');
+	} else if (seconds < 86401) {
+		ago = parseInt(seconds / 3600) + _('h ago');
+	} else {
+		ago = _('over a day ago');
+	}
+	var t = new Date(timestamp * 1000);
+	return t.toUTCString() + ' (' + ago + ')';
+}
+
+function generatePeerOption(key, title, value) {
+	return E('div', { 'class': 'cbi-value', 'style': 'padding: 0;' }, [
+		E('label', {
+			'class': 'cbi-value-title', 'style': 'font-weight: bold;'
+		}, title),
+		E('input', {
+			'class': 'cbi-input-text',
+			'data-name': key,
+			'style': 'border: none; float: left; width: 50%;',
+			'disabled': '',
+			'value': value
+		})
+	]);
+}
+
+function generatePeerTable(options, iconSrc) {
+	return E('div', { 'class': 'table cbi-section-table' }, [
+		E('div', { 'class': 'td' },
+			E('img', { 'src': iconSrc, 'class': 'tunnel-icon' })
+		),
+		E('div', { 'class': 'td peer-options' },
+			options.filter(function (option) {
+				return option[2] != null;
+			}).map(function (option) {
+				return generatePeerOption(...option);
```

Comment:
LuCI JS should stay at ES5 yet, can you replace the spread operator with function.apply? `generatePeerOption.apply(null, option)`